### PR TITLE
REPO: Clean up and adjustment to work with software-common

### PR DESCRIPTION
**Pull request by sentree**
**This pull request is part of a series of four. All four are on different repositories and it is crucial that they are merged at the same time to keep the install process working!**

These are the other pull requests to look at:
* [software-common #4](https://github.com/uf-mil/software-common/pull/4)
* [installer #2](https://github.com/uf-mil/installer/pull/2)
* [Navigator #192](https://github.com/uf-mil/Navigator/pull/192)

The general overview of this series can be found on the software-common repository pull request.

These are the Sub8 specific changes:
* Fixed the DNS hostname for the gumstix board in the navigation vessel

### CODE REVIEW

File: embedded/ActuatorBoard/readme.md [hunk 1]
```diff
@@ -1,27 +0,0 @@
-**Actuator Board**
```
Comment (mattlangford) [inline embedded/ActuatorBoard/readme.md:1]: Where is all this being moved to?

**Review by mattlangford — DISMISSED** [review]
Requesting to know where the embedded stuff is going. It needs to live somewhere

File: embedded/ActuatorBoard/readme.md (hunk 1 above)
Comment (sentree) [reply embedded/ActuatorBoard/readme.md:1]: @duggerd wants to store all of the electrical documentation and firmware on the MIL Confluence server.

Comment (duggerdufl) [reply embedded/ActuatorBoard/readme.md:1]: Electrical documentation is being kept on Confluence, the FW is being kept along side the PCB project in SVN.

**Review by DSsoto — APPROVED** [review]
Looks good, also @mattlangford's question was addressed so I'm approving.

**Pull request merged**